theme/theme.scss

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
```

Comment:
@oleq, do you think we could ship with such code styles? They fix `<code>` rendering (GH and Umberto use similar ones).